AGPs/AGP-Frame.md

```diff
@@ -0,0 +1,44 @@
+---
+AGP: ?
+Title: Flock Funding for Frame
+Author: Jordan Muir (@floating)
+Status: Pending
+Track: Finance
+Created: 2019/10/2
+---
+
+# AGP-Frame: Flock Funding for Frame
+## Address of the Transfer Recipient
+We request funds to be sent to our Aragon DAO: [`frame.aragonid.eth`](https://mainnet.aragon.org/#/frame.aragonid.eth) 0xe30b0d5168dfd8d92bc511e814de14e3757c775e
+
+## Amount of Transfer
+284,000 DAI
+
+50,000 ANT per year for the next 4 years with a 1 year cliff
+
+## Number of Transfers
+1
+
+## Purpose of Transfer
+Frame joining Flock allows us to **continue development and support of Aragon related initiatives**. We will continue **adding fundamental value to the Aragon ecosystem at the platform layer**. We'd like to embark on **deeper coordination within Aragon to provide support for any platform Aragon chooses to build on: desktop, mobile, Ethereum, Polkadot, IPFS, etc.** We’re looking forward to working with current and future Flock/Nest teams to fulfill their needs at this layer and to create a unified effort towards wider adoption. **We’ve made it our mission to keep our proposal focused and will over-deliver on our initiatives. We think our vision for adding value to Aragon is compelling and thoughtful. We hope others see the impact we'll have if included in Flock.**
+
+Frame being included in Flock gives Aragon the ability to create, iterate, improve and maintain a path for users to go from zero knowledge, to participating in a DAO. **This will be a very important feature for bigger teams, communities and organizations that want to start using Aragon. Large groups will be able to send out a simple link to all members, allowing everyone to seamlessly get up and running using their DAO without any external support.**
+
+ We think Aragon is a **fundamentally more efficient substrate to organize human effort**. As a team driven by the **acceleration of human innovation**, we’ll always support Aragon here. We believe **making the Aragon vision a reality will lead to massive jumps in the quality of life for all humans**.
+
+To cover our operation costs, Frame requests:
+* 284,000 DAI to cover 26 weeks of operating costs for 5 full-time employees and contractors
+* 50,000 ANT per year for the next 4 years as an alignment package (with a 1 year cliff)
```

Comment:
Comment: I find it a somewhat odd choice to have six months of operating costs but a four-year ANT package (with a one-year cliff). Can you explain the rationale here?

Reply:
I was told this is how incentive packages are done and can be revised and updated as we continue to participate in Flock.

Reply:
I think a one-off ANT package that can be updated over time is the route A1 and Autark went @john-light. That's also what Aragon Black planned to do before it withdrew its application.

Reply:
@LouisGrx  right now the assumption is that Finance track transactions are irreversible once confirmed here. Or at least, that is the norm some people seem to want to set. I think that it should either be made explicit that this ANT package can be changed later (by the AA? by ANT holders? both?) or that it should be written to have vesting over the full four years in case this Flock proposal is not renewed at a later date.

Reply:
Thanks for pointing that out. I think it makes sense to precise that it can be changed through an AGP indeed. 

In any case, the package presented here is closer to signalling, as Frame will have to propose another AGP in 6 monts (before the 1 year cliff is over). In 6 months the community will be able to reassess the package with new information in hand. If Frame isn't re-approved, my understanding is that no token would be vested at all because of the cliff.